src/cloudwatch/modules/metricdata.py

```diff
@@ -99,10 +100,14 @@ def _build_metric_name(self):
         return ".".join(name_builder)
     
     def _build_metric_dimensions(self):
-        dimensions = {
-              "Host" : self._get_host_dimension(),
-              "PluginInstance" : self._get_plugin_instance_dimension()
-              }
+        if self.config.DIMENSION_CONFIG_PATH != None:
```

Comment:
use "if self.config.DIMENSION_CONFIG_PATH:"

If the configuration file is empty. how to handle it?

Reply:
If the configuration file is empty, it will fall into the else case, which will use the original implementation to use the fixed dimension of "Host" and "PluginInstance".